guides/schematics.md

```diff
@@ -1,26 +1,30 @@
 Angular Material comes packaged with Angular CLI schematics to make
 creating Material applications easier.
 
-## Install Schematics
-Schematics come packaged with Angular Material, so once you have
-installed the npm package, they will be available via the Angular CLI.
+### Install Schematics
+Schematics come packaged with Angular Material. Once you install the npm package, they will be
+available through the Angular CLI.
 
-Using the command below will automatically install Angular Material for you
-and run the install schematic.
+Using the command below will install Angular Material, the [Component Dev Kit](../cdk) (CDK), and
```

Comment:
I was using the Getting Started Guide as an example. It uses `[theming guide](./theming.md)`. The Theming Guide also refers to `[theming-your-components.md](./theming-your-components.md)`.

It looks like the Creating a custom form field control Guide is the only one that uses a `material.angular.io` link rather than a relative URL.

Reply:
Perhaps the difference is referring to something outside of the `guides/` dir? As the Creating a custom form field control Guide is the only one that links to component docs via https://material.angular.io/components/form-field/api.